tests/integration/scheduler/test_eval.py

```diff
@@ -44,21 +43,25 @@
 DEFAULT_CONFIG['pki_dir'] = os.path.join(ROOT_DIR, 'pki')
```

Comment:
These kind of globals should be avoided.
`setUp` or `setUpClass` is the place to define them.